po/it.po

```diff
@@ -0,0 +1,209 @@
+"Last-Translator: Albano Battistella <piotrd>\n"
+"Language-Team: Italian <>\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
+"|| n%100>=20) ? 1 : 2);\n"
+
+#: data/org.freedesktop.Tuhi.appdata.xml.in.in:7
+#: data/org.freedesktop.Tuhi.desktop.in:3
+msgid "Tuhi"
+msgstr "Tuhi"
+
+#: data/org.freedesktop.Tuhi.appdata.xml.in.in:8
+#: data/org.freedesktop.Tuhi.desktop.in:4
+msgid "Utility to download drawings from the Wacom Ink range of devices"
+msgstr "Utilità per scaricare i disegni dalla gamma di dispositivi Wacom"
+
+#: data/org.freedesktop.Tuhi.appdata.xml.in.in:10
+msgid ""
+"Tuhi is a graphical user interface to download drawings stored on tablet "
+"devices from the Wacom Ink range, e.g. Intuos Pro Paper or Bamboo Slate."
+msgstr ""
+"Tuhi è un'interfaccia utente grafica per scaricare disegni archiviati su tablet"
+"dispositivi della gamma Wacom, ad esempio Intuos Pro Paper o Bamboo Slate."
+
+#: data/org.freedesktop.Tuhi.appdata.xml.in.in:15
+msgid ""
+"Tuhi requires Tuhi, the daemon to actually communicate with the devices. "
+"ThiGui is merely a front end to Tuhi, Tuhi must be installed and running "
+"when Tuhi is launched."
+msgstr ""
+Tuhi richiede Tuhi, il demone per comunicare effettivamente con i dispositivi. "
+"ThiGui è semplicemente un front-end di Tuhi, Tuhi deve essere installato e funzionante"
+"quando Tuhi viene lanciato."
+
+#: data/org.freedesktop.Tuhi.appdata.xml.in.in:32
+msgid "Tuhi's main window"
+msgstr "Finestra principale di Tuhi"
+
+#: data/org.freedesktop.Tuhi.appdata.xml.in.in:36
+msgid "Tuhi's main window (zoomed)"
+msgstr "Finestra principale di Tuhi (ingrandita)"
+
+#. Translators: Do NOT translate or transliterate this text (this is an icon file name)!
+#: data/org.freedesktop.Tuhi.desktop.in:7
+msgid "org.freedesktop.Tuhi"
+msgstr "org.freedesktop.Tuhi"
+
+#. Translators: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
+#: data/org.freedesktop.Tuhi.desktop.in:12
+msgid "tablet;wacom;ink;"
+msgstr "tablet;wacom;ink;"
+
+#: data/ui/AboutDialog.ui.in:13
+msgid "Visit Tuhi’s website"
+msgstr "Visita il sito web di Tuhi"
+
+#: data/ui/AppMenu.ui:5
+msgid "Portrait"
+msgstr "Ritratto"
+
+#: data/ui/AppMenu.ui:10
+msgid "Landscape"
+msgstr "Paesaggio"
+
+#: data/ui/AppMenu.ui:17
+msgid "Help"
+msgstr "Aiuto"
+
+#: data/ui/AppMenu.ui:21
+msgid "About"
+msgstr "Informazioni"
+
+#: data/ui/DrawingPerspective.ui:68
+msgid "Undo delete drawing"
+msgstr "Annulla l'eliminazione del disegno"
+
+#: data/ui/DrawingPerspective.ui:132
+msgid "Press the button on the device to synchronize drawings"
+msgstr "Premere il pulsante sul dispositivo per sincronizzare i disegni"
+
+#: data/ui/DrawingPerspective.ui:143 data/ui/SetupPerspective.ui:121
+msgid "page0"
+msgstr "pagina0"
```

Comment:
FTR, I filed #264 for this

Reply:
#264 is merged now, if you rebase on top of master the various `page0` ones should go away.